models/evc.py

```diff
@@ -399,6 +414,32 @@ def archive(self):
         """Archive this EVC on deletion."""
         self.archived = True
 
+    def _use_uni_vlan(self, uni: UNI):
+        """Use tags from UNI"""
+        if uni.user_tag is None:
+            return
+        tag = uni.user_tag.value
+        tag_type = uni.user_tag.tag_type.value
+        if isinstance(tag, int):
+            if not uni.interface.use_tags([tag, tag], tag_type):
+                intf = uni.interface.id
+                raise ValueError(f"Tag {tag} is not available in {intf}")
+            uni.interface.notify_interface_tags(self._controller)
+
+    def _disuse_uni_vlan(self, uni: UNI):
```

Comment:
At all call sites, the return type of `_disuse_uni_vlan` isn't being checked, it's only going to return a bool, should it also log as an error if at any point it tries to release a vlan it's not actually using? I'm thinking of a sanity sort of log four us in case it ever (accidentally) happens.

Reply:
Added `if` statement to catch a `False` from `Interface.make_tags_available()`. Logging warning: `f"Tag {tag} was already available in {intf}"`